specification/inference/_types/CommonTypes.ts

```diff
@@ -442,6 +442,149 @@ export enum AmazonBedrockServiceType {
   amazonbedrock
 }
 
+export class AmazonSageMakerServiceSettings {
+  /**
+   * A valid AWS access key that has permissions to use Amazon SageMaker and access to models for invoke requests.
+   */
+  access_key: string
+  /**
+   * The name of the SageMaker Endpoint.
+   * @ext_doc_id amazonsagemaker-invoke
+   */
+  endpoint_name: string
+  /**
+   * The API format to use when calling SageMaker.
+   * This will structure the payload when invoking the SageMaker endpoint.
+   */
+  api: AmazonSageMakerApi
+  /**
+   * The region that your endpoint or ARN is deployed in.
```

Comment:
It would be great to spell out the abbrev. I'm not sure that I suggested the right thing.
```suggestion
   * The region that your endpoint or Amazon Resource Name (ARN) is deployed in.
```